frontend/src/component/feature/FeatureView/FeatureOverview/ReleasePlan/ReleasePlan.tsx

```diff
@@ -378,10 +378,15 @@ export const ReleasePlan = ({
     );
 
     const handleSafeguardSubmit = (data: {
-        metric: string;
-        application: string;
-        aggregation: string;
-        comparison: string;
+        impactMetric: {
```

Comment:
this matches the structure that backend expects so when we add a hook it will just work